article/_posts/2019-10-23-Humain-Ai-Coord.md

```diff
@@ -0,0 +1,89 @@
+---
+layout: review
+title: "On the Utility of Learning about Humans for Human-AI Coordination"
+tags: reinforcement deep-learning
+author: "Antoine Théberge"
+cite:
+    authors: "Micah Carroll, Rohin Shah, Mark K. Ho, Thomas L. Griffiths, Sanjit A. Seshia, Pieter Abbeel, Anca Dragan"
+    title:   "On the Utility of Learning about Humans for Human-AI Coordination"
+    venue:   "NeurIPS 2019"
+pdf: ""
+---
+
+
+# Highlights
+- Self-play algorithms use (near) optimal solutions when playing together against humans
+- However, RL algorithms do not perform as well when playing _with_ humans
+- Training them to perform with humans makes them perform better with humans
+
+# Introduction
+
+> An increasingly effective way to tackle two-player games is to train an agent to play with a set of other AI agents, often past versions of itself. This powerful approach has resulted in impressive performance against human experts in games like Go, Quake, Dota, and Starcraft.
```

Comment:
@jhlegarreta 
> Don't the other examples mentioned use human-human examples (e.g. two human playing Go) to learn?
No, as stated here and in the parqagraph below, all "superhuman" AIs have been trained via self-play without any human demonstration or input.

Reply:
OK.